### PR TITLE
dts: riscv: mpfs_icicle devicetree cpu updates

### PR DESCRIPTION
**Pull request by con-pax**
Add the full complement of cpus available on the mpfs_icicle platform and
remove the old microsemi vendor compatible which is not correct for this platform.

### CODE REVIEW

**Review by galak — CHANGES_REQUESTED** [review]
Can  you update commit comment to describe what the change to clint is about as well.

**Comment by con-pax** [comment]
> Can you update commit comment to describe what the change to clint is about as well.
Hi @galak, will do and will push tomorrow, thanks!

**Comment by con-pax** [comment]
> Can you update commit comment to describe what the change to clint is about as well.

Hi @galak,
I have updated the commit message to describe the changes with the clint.
Cheers!

**Review by galak — APPROVED** [review]

**Comment by con-pax** [comment]
Hi @pgielda, @kgugala,

Could I request a review on this Devicetree change please?
Kind Regards,
Conor

File: dts/riscv/mpfs-icicle.dtsi [hunk 1]
```diff
@@ -62,8 +104,13 @@
 		clint: clint@2000000 {
 			compatible = "sifive,clint0";
 			interrupts-extended = <&hlic0 3 &hlic0 7
-					       &hlic1 3 &hlic1 7>;
-			interrupt-names = "soft0", "timer0", "soft1", "timer1";
+					       &hlic1 3 &hlic1 7
+					       &hlic2 3 &hlic2 7
+						   &hlic3 3 &hlic3 7
```
Comment (kgugala) [inline dts/riscv/mpfs-icicle.dtsi:109]: indentation here looks weird?

**Review by kgugala — APPROVED** [review]
one small formatting comment, otherwise LGTM

**Pull request merged**